tox_travis.py

```diff
@@ -67,10 +68,33 @@ def guess_python_env():
     return 'py{major}{minor}'.format(major=major, minor=minor)
 
 
+def get_default_envlist(version):
+    """Parse a default tox env based on the version.
+
+    The version comes from the ``TRAVIS_PYTHON_VERSION`` environment
+    variable. If that isn't set, then use sys.version_info to come
+    up with a reasonable default.
+    """
+    # If version isn't set, then guess from sys.version_info
+    if not version:
+        return guess_python_env()
+
+    # Assume single digit versions
+    match = re.match(r'^(\d)\.(\d)$', version)
```

Comment:
`LooseVersion` will give you the exact major and minor as distinct fields ; it will allow, and try to make sense of, other junk that you can ignore.
If you don't want to ignore the other junk, you can use the accompanying `StrictVersion`.